src/kernel.ts

```diff
@@ -610,9 +607,15 @@ class Kernel {
     this.disconnect();
     var ws_host_url = this._wsUrl + this._kernelUrl;
 
-    kernel_log.info("Starting WebSockets:", ws_host_url);
+    console.info("Starting WebSockets:", ws_host_url);
 
-    this._ws = new WebSocket(this.wsUrl);
+    if (typeof(WebSocket) !== 'undefined') {
+      this._ws = new WebSocket(this.wsUrl);
+    } else if (typeof(MozWebSocket) !== 'undefined') {
+      this._ws = MozWebSocket(this.wsUrl);
```

Comment:
It was in the old code, and I assumed it was used by Firefox, but the console in 38 says otherwise...